java/src/main/java/com/powsybl/python/SensitivityAnalysisContext.java

```diff
@@ -132,7 +132,15 @@ void setFactorMatrix(List<String> branchsIds, List<String> injectionsOrTransfosI
         return factors;
     }
 
-    SensitivityAnalysisResultContextV1 runV1(Network network, LoadFlowParameters loadFlowParameters, String provider) {
+    SensitivityAnalysisResultContext run(Network network, LoadFlowParameters loadFlowParameters, String provider) {
+        if ("OpenLoadFlow".equals(provider)) {
```

Comment:
We should be consistent with the actual provider name which is "OpenSensitivityAnalysis"